app/components/rl-dropdown.js

```diff
@@ -15,26 +14,17 @@ export default Component.extend({
 
   isExpanded: alias('dropdownContainer.dropdownExpanded'),
 
-  closeOnChildClick: false,
-
-  propagateClicks: true,
-
   click(event) {
-    let closeOnChildClick = this.closeOnChildClick;
-    let propagateClicks = this.propagateClicks;
-    let $target = $(event.target);
-    let $c = this.$();
+    let closeOnChildClick = 'a:link';
+    let $target = event.target;
+    let $c = this.element;
```

Comment:
Both of them are not jQuery objects anymore. It would be better to remove `$` from the names.

Reply:
I left them as is to minimize the changes in the code. Also, this code will likely be changed or moved in a subsequent pass.